client/src/modules/user/AuthBox/components/SignInForm.jsx

```diff
@@ -128,26 +122,20 @@ class SignInForm extends PureComponent {
     );
   };
 
-  renderForgotPassword = () => {
-    const {
-      intl: { formatMessage }
-    } = this.props;
-
-    return (
-      <p className="sign-in__forgot-password">
-        <FormattedMessage
-          id="authorization.forgot-password.question"
-          values={{
-            link: (
-              <Link className="sign-in__link" to="/reset-password">
-                {formatMessage({ id: 'authorization.forgot-password' })}
-              </Link>
-            )
-          }}
-        />
-      </p>
-    );
-  };
+  renderForgotPassword = () => (
```

Comment:
Please see my comment about the extraction of subcomponents.

Reply:
added to JIRA under EOC_407 task